test/container/BUILD

```diff
@@ -90,7 +90,7 @@ install_pkgs(
     name = "docker-prereq-test",
     image_tar = "@debian8//image",
     installables_tar = ":docker-prereq-packages.tar",
-    output_image_name = "bazel/test/container:docker-prereq-test",
+    output_image_name = "docker-prereq-test:intermediate",
```

Comment:
I think tagging them as latest should be good. They are not really intermediate images.

Reply:
Fixed